app/forms.go

```diff
@@ -29,7 +29,9 @@ var Forms = []forms.Form{
 
 		Formatter: forms.DefaultFormatter{Color: 0x34eb5b}.Fields(
 			"why", "whyNot", "link",
-		),
+		).
+			SetFieldName("whyNot", "Why Not?").
+			SetFieldName("why", "Why?"),
```

Comment:
what's up with the wakkie-nu-nu formatting my man

Reply:
ask `go fmt` 
maybe this would look less weird?
```go
Formatter: forms.DefaultFormatter{Color: 0x34eb5b}.
			Fields("why", "whyNot", "link").
			SetFieldName("whyNot", "Why Not?").
			SetFieldName("why", "Why?"),
```